views/keyword/_url_result_report.html

```diff
@@ -0,0 +1,15 @@
+<div class="search-url row">
+  <div class="search-url--result col-lg-6 col-sm-12">
+    <h4 class="search-url__title">Search Results Urls</h4>
+    {{ range $url := .KeywordPresenter.ResultsUrls }}
+      <a  href="{{ $url }}" class="search-url__item d-inline-block text-truncate" target="_blank">{{ $url }}</a>
+    {{ end }}
+  </div>
+
+  <div class="search-url--advertiser col-lg-6 col-sm-12">
```

Comment:
`--result` and `--advertiser` seems to be Element (in BEM) under the block `search-url` 🤔 So it could be `search-url__panel` and then have different variations for `result` and `advertiser` with `search-url__panel--result` and `search-url__panel--advertiser`. You can recheck [the official documentation](https://en.bem.info/methodology/quick-start/#element) if you are unsure.

Reply:
[Getbem](http://getbem.com/naming/) mentioned `modifier` can be used directly on block also(without element).With the element it goes a bit nested nested like bellow ...
```html
<div class="search-url">
    <div class="search-url__panel">
        <div class="search-url__panel--result">
         ...
        </div>
        <div class="search-url__panel--advertiser">
        ...
        </div>
    </div>
</div>
```  
And I need to apply `make-row` on `search-url__panel` instead of `search-url`. What is your thought?

![Screen Shot 2021-03-22 at 4 12 55 PM](https://user-images.githubusercontent.com/6076762/111966530-9a163a00-8b29-11eb-9d34-38f82f933dbe.png)

Reply:
Indeed, the modifier can be used on the block element too. However in this case, I don't see the need to use a modifier at all. Both `search-url__panel` and `search-url__panel` seem to be identical in terms of styling. So the following would be sufficient:

```html
<div class="search-url">
    <div class="search-url__panel-container">
        <div class="search-url__panel">
         ...
        </div>
        <div class="search-url__panel">
        ...
        </div>
    </div>
</div>
```

Reply:
Updated in https://github.com/junan/google-scraper/pull/39/commits/d8712c11a2a872bbd14bce5ed3968aa14189ce6f